website/src/components/events/event-list-item.js

```diff
@@ -85,25 +85,35 @@ export default function EventListItem(props) {
     eventType = "Community Event"
   }
 
+  // Gatsby seems to hijack all <a> tags and, if they do not begin with http://
+  // or https://, appends the path to the domain instead of setting the URL as
+  function checkLink(param) {
```

Comment:
Seeing this in two places makes me think it might be nice to have this as a function we could export to any component throughout the site, but we can save that for a refactor ticket.

Reply:
just because you mentioned about `export`, I just move the function into `index.js` file
or if its not a right place we can move it to somewhere else as per future need.